src/commands/run.py

```diff
@@ -20,21 +20,31 @@ def add_commands(appliance):
     def run():
         pass
 
+    run_options = {
+        **cli_utils.hash__node__group,
+        ('--yes', '-y'): {
+            'help': 'Skip the confirmation prompt',
+            'is_flag': True
+        }
+    }
+
     runner_cmd = {
         'help': Config.help,
         'arguments': [['remote_arguments']], # [[]]: Optional Arg
-        'options': cli_utils.hash__node__group
+        'options': run_options
     }
     runner_group = {
         'help': (lambda names: "Run tools in {}".format(' '.join(names))),
         'invoke_without_command': True,
-        'options': cli_utils.hash__node__group
+        'options': run_options
     }
 
     @command_creator.tools(run, command = runner_cmd, group = runner_group)
     @cli_utils.with__node__group
     @cli_utils.ignore_parent_commands
-    def runner(_ctx, configs, argv, _, nodes):
+    def runner(_ctx, configs, argv, options, nodes):
+        if not options['yes'].value:
+            if not get_confirmation(configs, nodes): return
```

Comment:
This can be simplified to:
```
if not (options['yes'].value or get_confirmation(configs, nodes)):
```

Reply:
done here https://github.com/alces-software/adminware/pull/166/commits/72d9e4e208b4beed406ee0c491c6246d8c3ddecb